__tests__/app.test.js

```diff
@@ -81,12 +81,32 @@ describe("GET /api/articles/:article_id", () => {
 });
 
 describe("GET /api/articles", () => {
-	it("status 200: return an array of objects", () => {
+	it("should return an array", () => {
 		return request(app)
 			.get("/api/articles")
 			.expect(200)
 			.then(({ body: { articles } }) => {
-				expect(articles).toBeInstanceOf(Array);
+				expect(Array.isArray(articles)).toBe(true);
+			});
+	});
+	it("status 200: return an array of objects that have a comment_count as an added column", () => {
+		return request(app)
+			.get("/api/articles")
+			.expect(200)
+			.then(({ body: { articles } }) => {
+				expect(articles).not.toHaveLength(0);
+				articles.forEach((article) => {
+					expect(article).toMatchObject({
+						author: expect.any(String),
+						title: expect.any(String),
+						article_id: expect.any(Number),
+						topic: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number),
+						article_img_url: expect.any(String),
+						comment_count: expect.any(String),
+					});
+				});
 			});
 	});
 });
```

Comment:
Please write an additional test to check that the default sort & order is : created_at, desc